tests/test_return_value.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+
+@pytest.mark.benchmark
+def test_return_value(benchmark):
+    def calculate():
+        return 1 + 1
+
+    result = benchmark(calculate)
+
+    assert result == 2
```

Comment:
Since we're testing the pytest plugin, the easiest way to run pytest on this sample would be to have a similar approach as in this one:
https://github.com/CodSpeedHQ/pytest-codspeed/blob/48e42286995b02761d922fed9d88d758c0636a4e/tests/test_pytest_plugin.py#L252-L265

And just testing `result.ret == 0` will be perfect

Reply:
updated!